src/main/java/com/minecraftabnormals/upgrade_aquatic/core/events/EntityEvents.java

```diff
@@ -181,6 +184,15 @@ public static void onPlayerMount(EntityMountEvent event) {
 		}
 	}
 	
+    @SubscribeEvent
+    public static void onDrownedPoseChange(EntityEvent.EyeHeight event) {
```

Comment:
You call `event.getEntity()` multiple times. This is redundant, it'd be more clean and performant to have it be stored as a local variable.